GitHubData/Exceptions/RateLimitException.cs

```diff
@@ -0,0 +1,44 @@
+// <copyright file="RateLimitException.cs" company="KinsonDigital">
+// Copyright (c) KinsonDigital. All rights reserved.
+// </copyright>
+
+namespace GitHubData.Exceptions;
+
+// TODO: Create tests
+
+/// <summary>
+/// Occurs the GitHub request rate limit has been reached.
```

Comment:
Refactor code docs to below:

```suggestion
/// Occurs when the GitHub request rate limit has been reached.
```